crates/router_env/Cargo.toml

```
@@ -24,7 +24,7 @@ tracing = { version = "=0.1.36" }
 tracing-actix-web = { version = "0.7.4", features = ["opentelemetry_0_18"], optional = true }
```

Optionally, you can also update this:

```suggestion
tracing-actix-web = { version = "0.7.5", features = ["opentelemetry_0_19"], optional = true }
```

---

changed